docs/src/genmd-filter

```diff
@@ -37,6 +37,10 @@ def main
         lines = read_until_genmd_eof(input_handle)
         write_card([], lines, output_handle)
 
+      elsif content_line =~ /^GENMD-CARDIFY-HIGHLIGHT-ALL$/
```

Comment:
nice 🎆

Reply:
That's the code that worries me most, as this is actual running code that can fail a build. And I didn't tested that modification. But I hope it is easy enough to work at first try .-)